packages/react-components/src/components/Picker/PickerList.tsx

```diff
@@ -166,6 +187,29 @@ export const PickerList: React.FC<IPickerListProps> = ({
           );
         }
 
+        if (item.key === 'select-all') {
+          return (
+            <li
+              ref={(element) => {
+                if (currentItemKey === item.key) {
+                  element?.scrollIntoView({ block: 'nearest' });
+                }
+              }}
+              role="option"
+              aria-current={currentItemKey === item.key}
+              id={item.key}
+              key={item.key}
+              className={cx(
+                styles[itemClassName],
+                styles[`${itemClassName}--select-all`]
+              )}
+              onClick={handleOnSelectAllClick}
+            >
+              Select all
```

Comment:
Could be a const

Reply:
Additionally, I believe developers should have control over this text. For example if used in non-English environment